pkgs/development/python-modules/opensfm/default.nix

```diff
@@ -64,16 +66,25 @@ buildPythonPackage rec {
     echo 'feature_type: SIFT' >> data/berlin/config.yaml
     echo 'feature_type: HAHOG' >> data/lund/config.yaml
 
+    # make opensfm correctly import glog headers
+    substituteInPlace opensfm/src/CMakeLists.txt \
+      --replace \
+        'find_package(glog REQUIRED)' \
+        "add_compile_definitions(GLOG_USE_GLOG_EXPORT)"$'\n'"find_package(glog REQUIRED)"
+
     sed -i -e 's/^.*BuildDoc.*$//' setup.py
   '';
 
+  build-system = [ setuptools ];
+
   nativeBuildInputs = [
     cmake
     pkg-config
     sphinx
   ];
+
   buildInputs = [
-    ceres'
+    (lib.getDev ceres-solver)
```

Comment:
Actually, even this shouldn't be necessary, because `mkDerivation` applies `getDev` to `*Inputs`